### PR TITLE
SW-5410 Make the add species to projects modal a multi select modal

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2656?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2656** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2656?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2647** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2647?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2628** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2628?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 3 other dependent PRs ([#2629](https://github.com/terraware/terraware-web/pull/2629) <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2629?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#2632](https://github.com/terraware/terraware-web/pull/2632) <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2632?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#2639](https://github.com/terraware/terraware-web/pull/2639) <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2639?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/Species/AddToProjectModal.tsx [hunk 1]
```diff
@@ -10,22 +10,37 @@ import { Project } from 'src/types/Project';
...
 
 export default function AddToProjectModal(props: AddToProjectModalProps): JSX.Element {
   const { onClose, onSubmit, projects } = props;
   const theme = useTheme();
-  const [selectedProject, setSelectedProject] = useState<Project>();
+
+  const [selectedProjectIds, setSelectedProjectIds] = useState<number[]>([]);
+
+  const handleOnAdd = useCallback((value: number) => {
+    console.log('value', value);
```
Comment (joshbuchea) [inline src/scenes/Species/AddToProjectModal.tsx:24]: Remove `console.log`?

**Review by joshbuchea — APPROVED** [review]
Nice!

**Comment by nickgraz** [comment]
@joshbuchea would you mind taking a look at https://github.com/terraware/terraware-web/pull/2647 also?

**Comment by joshbuchea** [comment]
@nickgraz you got it

**Comment by nickgraz** [comment]
### Merge activity

* **May 22, 6:23 PM EDT**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2656).

**Pull request merged**